lib/utils.js

```diff
@@ -1,6 +1,9 @@
 
 "use strict";
 var crypto = require('crypto');
+var moment = require('moment');
+var moment = require('moment-timezone');
```

Comment:
it's confusing to have multiple `var moment` declarations.  Can you ditch the `var moment = ` part for `require('moment')`?